### PR TITLE
Add blog post about recent Clippy team changes

### PR DESCRIPTION
**Pull request by flip1995**
We changed the Clippy team quite a bit recently with 3 new team additions. We want to get this a little bit more visibility and decided to write up a blog post, like other teams do.

cc @rust-lang/clippy

### CODE REVIEW

File: posts/inside-rust/2022-06-30-clippy-team-changes.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+With the new additions to the team, we should be able to handle our long PR
+queue better. We currently have over 40 open PRs, which is a good sign for the
+health of the project, but doesn't help when we don't have the reviewers to
+merge them. Now we do! :tada:
+
+## Alumni
+
+At the same time, we recently granted some of our oldest members the well
+deserved alumni status. We want to thank [Pascal
```
Comment (AngelOnFira) [inline posts/inside-rust/2022-06-30-clippy-team-changes.md:38]:
```suggestion
At the same time, we recently granted some of our oldest members the
well-deserved alumni status. We want to thank [Pascal
```

**Review by Manishearth — APPROVED** [review]
r? @rylev

File: posts/inside-rust/2022-06-30-clippy-team-changes.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+We also want to thank [Philipp Hansch](https://github.com/phansch) for their
+contributions to Clippy and all of the review work they've done over the years.
+To this day phansch is one of the most active Clippy contributors (measured by
+the number of commits).
+
+And finally, thank you to [mikerite](https://github.com/mikerite), which we are
+lucky to still have as a contributor and only have to say goodbye to as an
+active reviewer/team member. We're looking forward to continue working with
+them!
```
Comment (AngelOnFira) [inline posts/inside-rust/2022-06-30-clippy-team-changes.md:52]: I think this paragraph is a bit confusing. I think this conveys the same intention, but might still need some work.

```suggestion
And finally, thank you to [mikerite](https://github.com/mikerite), who will
be stepping down from an active reviewer/team member position, but will
be staying on as a contributor. We're looking forward to continue working
with them!
```

**Review by rylev — CHANGES_REQUESTED** [review]
Looks great! We might have another post in the docket for today (though on the main blog and not the inside-rust blog). We may want to hold off until 04-July to post. In the meantime, there's a few nits to sort through.

**Comment by flip1995** [comment]
Just noticed that this is PR #999. Sooo close :smiling_face_with_tear:

**Comment by rylev** [comment]
@flip1995 sorry for the delay - I was on holiday last week. Does tomorrow (2022-07-12) work?

**Comment by flip1995** [comment]
Works for me. I squashed+rebased the post and updated the date to tomorrow :+1:

**Comment by flip1995** [comment]
ping @rylev bumped the date to today. Yesterday this would've interfered with the core team update. I would like to get this merged rather sooner than later now.

**Review by rylev — APPROVED** [review]

**Comment by rylev** [comment]
@flip1995 it has been merged! Thanks for your patience!

**Pull request merged**